### PR TITLE
[yamahareceiver] Migrate to bnd build

### PR DESCRIPTION
**Pull request by davidgraeff**
As usual I'm not only doing a migration but also modernization of migrated code.
The first commit is the migration.
The second commit contains Eclipse autoformat changes.
The 3rd commit makes the code use the `getServices()` pattern and not the deprecated bundeServices field.

### CODE REVIEW

**Comment by davidgraeff** [comment]
@J-N-K But this is the official ESH formatting. The binding is just soo old that it doesn't comply to the current formatter settings

**Comment by davidgraeff** [comment]
I'm not happy about the xml schemata changes

**Comment by martinvw** [comment]
> I'm not happy about the xml schemata changes

@davidgraeff what do you mean, does it refer to changes in this PR or other changes?

**Comment by davidgraeff** [comment]
Other changes. I need to adapt the pr

**Comment by davidgraeff** [comment]
@martinvw When you have time :) Most code changes are auto-formatting, as this binding was never completely formatted with the ESH profile. The factory received some changes, because of the different discovery service handling.

File: bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/config/YamahaZoneConfig.java [hunk 1]
```diff
@@ -38,7 +42,7 @@
      */
     private float volumeDbMax = 12f; // 12.0 dB
 
-    public YamahaReceiverBindingConstants.Zone getZone() {
+    public YamahaReceiverBindingConstants.@Nullable Zone getZone() {
```
Comment (J-N-K) [inline bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/config/YamahaZoneConfig.java]: That looks weird. Any possibility to move the `@Nullable` to the front?

Comment (davidgraeff) [reply bundles/org.openhab.binding.yamahareceiver/src/main/java/org/openhab/binding/yamahareceiver/internal/config/YamahaZoneConfig.java]: The only way would be to statically import Zone from YamahaReceiverBindingConstants I guess

**Pull request merged**